opensubtitles/common.go

```diff
@@ -0,0 +1,153 @@
+package opensubtitles
+
+import "time"
+
+//Subtitles
```

Comment:
Codacy found an issue: [comment on exported type Subtitles should be of the form "Subtitles ..." (with optional leading article)](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)